src/controllers/WhoWasIllController.ts

```diff
@@ -0,0 +1,18 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller, httpGet } from 'inversify-express-utils';
+
+import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { Feature } from 'app/utils/Feature';
+import { WHO_WAS_ILL_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'illness/who-was-ill';
+
+@controller(WHO_WAS_ILL_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON), SessionMiddleware)
+export class WhoWasIllController extends BaseAsyncHttpController {
```

Comment:
Why are we not using base controller here?

Reply:
Just wanted to knock a quick and easy controller up with minimal code, as it isn't included in this story. I'm hoping to pick up the relevant story next so will implement the controller properly (and using BaseController) then.